src/config/constants.ts

```diff
@@ -134,3 +134,22 @@ export const NYPL_LOCATIONS = {
     address: "476 Fifth Avenue (42nd St and Fifth Ave)",
   },
 }
+
+export const availabilityKeys = {
+  // anything not covered by the cases below is EDGE_CASE
+  EDGE_CASE: "edgeCase",
```

Comment:
Can you add a test for this?

Reply:
I don't actually know that this case is possible, so I forced it in the test as `item.availability.key === "edgeCase"`